### PR TITLE
add user controller

### PR DESCRIPTION
**Pull request by Yvvie**

### CODE REVIEW

**Comment by Yvvie** [comment]
Add user Controller Schema

File: models/userModel.js [hunk 1]
```diff
@@ -1,50 +1,34 @@
...
 			type: String,
 			required: true,
-			// unique validacija kad nebutu issaugotas objektas duomenu bazej su tokiu paciu email value
 			unique: true,
 		},
 		password: {
 			type: String,
 			required: true,
 		},
-		createdAt: {
```
Comment (dzonasm) [inline models/userModel.js:16]: please keep this

**Review by dzonasm — CHANGES_REQUESTED** [review]
please apply changes

**Review by dzonasm — APPROVED** [review]
looks good, merging

**Pull request merged**